friendly-captcha/modules/divi/script.js

```diff
@@ -0,0 +1,59 @@
+(function () {
```

Comment:
We have a script that does essentially the same already in `public/mutation-observer.js` which is opt-in in the plugin settings. Maybe we can just re-use that and always load it when divi forms is enabled?

Reply:
Good pointer, will check.

Reply:
Now using the mutation-observer from the "core" - I also made some improvements with the loading order.